assets/js/components/ClusterDetails/ClusterInfoBox.test.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import ClusterInfoBox from './ClusterInfoBox';
+
+describe('Provider Label', () => {
+  it('should display AWS as the provider and HA Scenario: "HANA scale-up"', () => {
```

Comment:
Just now thinking about this.
Is it possible to create tests with data coming from a loop?
Something like:
```
[{haScenario: "hana_scale_up", haScenarioText: "HANA Scale-up", provider: "aws", providerText: "AWS"},{{haScenario: "hana_scale_out", haScenarioText: "HANA Scale-out",...}].forEach(({haScenario: haScenario, haScenarioText: haScenarioText) => {
        it(`should display ${providerText} as the provider and HA Scenario: ${haScenarioText}`, () => {
              render(<ClusterInfoBox haScenario="hana_scale_up" provider="aws" />);
              expect(screen.getAllByText(/AWS/)).toBeTruthy();
...
```

This way we can easily add all the scenario (if we want) without adding more code

Reply:
> Is it possible to create tests with data coming from a loop?

I don't know actually? We need to try :D